app/src/main/java/com/example/cristiana/weatherapp/DayDetailsActivity.java

```diff
@@ -0,0 +1,48 @@
+package com.example.cristiana.weatherapp;
+
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.TextView;
+
+import com.example.cristiana.weatherapp.model.Forecast;
+import com.example.cristiana.weatherapp.model.Main;
+import com.example.cristiana.weatherapp.model.Weather;
+import com.example.cristiana.weatherapp.model.Wind;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DayDetailsActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        updateInfo();
+
+    }
+
+    public void updateInfo() {
+        Forecast forecast = new Forecast();
+        Main main =  new Main();
+        Weather weather = new Weather();
+        Wind wind = new Wind();
+        List<Weather> weatherList = new ArrayList<>();
+
+        main.setTemp(getIntent().getDoubleExtra("temperature", 0));
```

Comment:
You can make the Forecast to implement Parcelable to do your serialization automatically (don't forget to make all of its member classes Parcelable before that)